src/app-directory/README.md

```diff
@@ -1,7 +1,7 @@
 # Application Directory API -- THIS IS NOT A RATIFIED FDC3 API or SPECIFICATION
 
 **This is NOT a ratified API or specification but rather a tool to evaluate both the use cases and technology implementations.**
-[View FULL SPECIFICATION HERE](https://fdc3-app-directory.finos.org/)
+[View FULL FDC3 SPECIFICATION HERE](https://fdc3.finos.org/)
```

Comment:
Ooh good catch, I never really looked at this file.

Yes it should probably point to https://fdc3.finos.org/docs/1.0/appd-spec.